rpc/client/client_test.go

```diff
@@ -214,6 +214,93 @@ func TestGetBlock(t *testing.T) {
 	require.NoError(err)
 }
 
+func TestUnconfirmedTxs(t *testing.T) {
```

Comment:
General remark: I think this could be a table driven test. But this is fine for now.

Reply:
Good point, refactored.